docs/distribution/distribution.md

```diff
@@ -202,20 +204,19 @@ but the following are some guidelines:
 3. Packages that the compiler relies on, e.g. compile error definitions, stack
    traces etc.
 
-### Resolvers
+## Enso Home Layout
 
-**Note** This system is not implemented yet.
+The location called in some places `<ENSO_HOME>` is the place where user's
+projects and similar files are stored. Currently it is specified to always be
+`$HOME/enso`.
 
-A resolver is a manifest containing library versions that are automatically
-available for import in any project using the resolver.
+It has the following structure:
 
-Example contents of a resolver file are as follows:
-
-```yaml
-enso-version: 1.0.7
-libraries:
-  - name: Base
-    version: 1.0.0
-  - name: Http
-    version: 5.3.5
+```
+<ENSO_HOME>
+├── projects             # Contains all user projects.
+├── libraries            # Contains all local libraries that can be edited by the user.
```

Comment:
This will be refined in #1772, but I think the idea is to keep them in `ENSO_DATA_DIRECTORY/lib` and this is (maybe a little indirectly) mentioned in `Installed Enso Distribution Layout`.

Reply:
Oh no I actually noticed that it is stated rather directly:
```
ENSO_DATA_DIRECTORY
(...)
├── lib                     # Contains sources of downloaded libraries.
│   └── Standard            # Each prefix (usually corresponding to the author) is placed in a separate directory.
│       └── Dataframe       # Each library may be stored in multiple versions.
│           └── 1.7.0       # Each version contains a standard Enso package.
│               ├── package.yaml
│               └── src
│                   ├── List.enso
│                   ├── Number.enso
│                   └── Text.enso
```